src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContentProvider.java

```diff
@@ -23,7 +25,16 @@ public AmountContentProvider(InterestContentProvider interestContentProvider) {
     }
 
     public AmountContent create(Claim claim) {
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
         BigDecimal claimAmount = ((AmountBreakDown) claim.getClaimData().getAmount()).getTotalAmount();
+
+        BigDecimal settledClaimAmount = claimAmount;
+        if (claim.getResponse().isPresent()
+            && claim.getResponse().get() instanceof PartAdmissionResponse
+            && countyCourtJudgment.isSettledLessThanClaimAmount()) {
```

Comment:
What is the purpose of that boolean flag? An amount will be lower every time part admission exists so I find having `settledLessThanClaimAmount` a bit useless. Is there anything I missed?

Reply:
front end change checks for `claim.response && claim.response.responseType === ResponseType.PART_ADMISSION
      && draft.settleAdmitted && draft.settleAdmitted.admitted === YesNoOption.YES`

Reply:
I think passing this boolean from frontend is ambiguous and admitted amount can be derived from response itself. If we were to pass anything from the frontend I would suggest passing claim amount that is a subject of CCJ, not boolean flag.

Reply:
@ckalista you might want to have a look and give you opinion. I advised against that boolean flag but at the same time gave @dharmendrak choice in case he wanted to bring existing implementation to prod.

Reply:
Seeing the claimant response journey - claimant can still go ahead with CCJ by admission on full claim amount by disagreeing with Defendant part admitted amount whilst formalising repayment plan and in that we needed to store claimant's response whether he/she has agreed with Def or not. Secondly, admitted amount on  CCJ would be either full claim amount or defence part admitted amount which we already have in back end. I'm happy with passing admittedAmount instead of boolean but we would require migration script on backend to back fill all exiting CCJs by default objects.

Reply:
Spoken to @dharmendrak and seems like there's a bug where I can still go ahead with CCJ by rejecting Def's part admitted amount by going backward/forward on task list. Demonstrated bug to @dharmendrak.